search-app/src/cmr/search/api/concepts_search.clj

```diff
@@ -368,7 +371,8 @@
 
 (def search-routes
   "Routes for /search/granules, /search/collections, etc."
-  (context ["/:path-w-extension" :path-w-extension #"(?:(?:granules)|(?:collections)|(?:variables)|(?:subscriptions)|(?:tools)|(?:services))(?:\..+)?"] [path-w-extension]
+  ;; TODO: Generic work: Think about putting this into a config file and creating the reg-ex from there.
```

Comment:
Please add to a sub-task or checklist item on the parent ticket and remove the TODO

Reply:
https://bugs.earthdata.nasa.gov/browse/CMR-8619